### PR TITLE
chore: proper exports inside chain-adapters package

### LINKED ISSUE #462: Export everything correctly from lib
### Overview

Currently, the `web` repo is consuming imports from this `lib` repo, but importing directly from the `dist` directory of bunch of packages. This isn't correct and needs fixing.

### Reference

Anything coming from a `@shapeshiftoss/*` package should be exported correctly in lib, and not imported via the `/dist/` folder in web

```bash
➜  web git:(develop) rg /dist/
src/pages/TransactionHistory/TransactionHistoryFilter.tsx
16:import { TradeType, TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/pages/TransactionHistory/DownloadButton.tsx
2:import { TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/pages/TransactionHistory/components/DatePicker/DatePicker.tsx
1:import 'react-datepicker/dist/react-datepicker.css'

src/components/Modals/Send/views/Address.tsx
13:import { ChainAdapter as EthereumChainAdapter } from '@shapeshiftoss/chain-adapters/dist/ethereum/EthereumChainAdapter'

src/components/TransactionHistoryRows/TransactionRow.tsx
3:import { TradeType, TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/components/TransactionHistoryRows/TransactionGenericRow.tsx
3:import { TradeType, TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/components/Transactions/TransactionRow.tsx
3:import { TradeType, TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/features/defi/helpers/normalizeOpportunity.tsx
2:import { AssetNamespace } from '@shapeshiftoss/caip/dist/caip19/caip19'

src/test/mocks/txs.ts
2:import { TradeType } from '@shapeshiftoss/types/dist/chain-adapters'

src/plugins/cosmos/index.tsx
1:import { ChainAdapter as CosmosChainAdapter } from '@shapeshiftoss/chain-adapters/dist/cosmossdk/cosmos'
2:import { ChainAdapter as OsmosisChainAdapter } from '@shapeshiftoss/chain-adapters/dist/cosmossdk/osmosis'

src/hooks/useBalanceChartData/useBalanceChartData.ts
3:import { TxType } from '@shapeshiftoss/types/dist/chain-adapters'

src/state/slices/txHistorySlice/txHistorySlice.ts
2:import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/dist/query'

src/state/slices/assetsSlice/assetsSlice.ts
2:import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/dist/query/react'

src/state/slices/marketDataSlice/marketDataSlice.tsx
2:import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/dist/query/react'

src/hooks/useTxDetails/useTxDetails.ts
2:import { TradeType, TxTransfer, TxType } from '@shapeshiftoss/types/dist/chain-adapters'

cypress/factories/ethereum/transactions.ts
1:import { TxHistory } from '@shapeshiftoss/unchained-client/dist/generated/ethereum'

cypress/factories/ethereum/account.ts
1:import { EthereumAccount } from '@shapeshiftoss/unchained-client/dist/generated/ethereum'

cypress/factories/bitcoin/account-xpub.ts
1:import { BitcoinAccount } from '@shapeshiftoss/unchained-client/dist/generated/bitcoin'

cypress/factories/bitcoin/account.ts
1:import { BitcoinAccount } from '@shapeshiftoss/unchained-client/dist/generated/bitcoin'

cypress/factories/bitcoin/transactions.ts
1:import { TxHistory } from '@shapeshiftoss/unchained-client/dist/generated/bitcoin'

cypress/fixtures/wallet.ts
1:import { FlattenedJWE } from 'jose/dist/types/types'
```

### Acceptance Criteria

This bounty will require two PRs

* Make a lib PR fixing all these exports
* Make a corresponding web PR updating the imports to use the new paths

### Need By Date

_No response_

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by pbteja1998**
fixes #462

### CODE REVIEW

File: packages/chain-adapters/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
 export * from './ChainAdapterManager'
 export * from './api'
 export * from './utils'
+export * as cosmossdk from './cosmossdk'
+export * as ethereum from './ethereum'
```
Comment (kaladinlight) [inline packages/chain-adapters/src/index.ts:5]: Might as well add a bitcoin export here as well in case we need it in the future.

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
thank @pbteja1998 - can you please implement kevin's suggestion for future proofing?

**Review by kaladinlight — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
@pbteja1998 merged - can you please update the web PR with the new published packages in a few minutes after CI publishes them to npm? ty :)

**Comment by pbteja1998** [comment]
@0xdef1cafe sure, will update it.

**Pull request merged**